docs/metrics/ahas_sentinel.md

```diff
@@ -0,0 +1,21 @@
+## AHAS Sentinel External metrics
+
+#### Global Params
+
+All metrics need the global params.
+
+| Global params       | Description              | Example            | Required | Default value |
+| ------------------- | ------------------------ | ------------------ | -------- | ------------- | 
+| `ahas.sentinel.app.name` | The name of your service in AHAS | sentinel-console | True |  |
+| `ahas.sentinel.namespace` | The namespace of your service in AHAS | staging | False | default |
+| `ahas.sentinel.stat.period` | The statistic period of request count (in second) | 5 | False | 1 |
```

Comment:
Is default value "1" large enough?

Reply:
Pls change to `ahas.sentinel.period`  and `ahas.sentinel.app`. Because the dimensions have different meanings.

Reply:
Pls add some summary comment

Reply:
Updated, please check.

The default interval has been updated to 10s.